### PR TITLE
Replace oc cluster up with minikube for PR jenkins jobs

### PR DESCRIPTION
**Pull request by Frawless**
Signed-off-by: Jakub Stejskal <xstejs24@gmail.com>

### Type of change

- Enhancement / new feature

### Description

Since we are going to drop support for kube version < 1.15.0, we have to replace `oc cluster up` with something different. 

This PR removes `oc cluster up` from PR jobs and run instead of it minikube. Users can specify kube version via `kube_version=1.19.0` string in the triggering comment. The default value is still 1.15.0, for azure and jenkins as well.

I left in code a few comments on how to allow NP and how to deal with issues caused by it. I would leave it for another PR when https://github.com/strimzi/strimzi-kafka-operator/pull/4092 will be merged.

### Checklist

- [x] Make sure all tests pass
- [x] Update documentation

### CODE REVIEW

**Comment by Frawless** [comment]
@strimzi-ci run tests profile=smoke test_only

**Comment by Frawless** [comment]
@strimzi-ci run tests profile=smoke test_only

**Comment by Frawless** [comment]
@strimzi-ci run tests profile=smoke test_only

**Comment by Frawless** [comment]
@strimzi-ci run tests profile=smoke test_only

**Comment by Frawless** [comment]
@strimzi-ci run tests profile=smoke test_only kube_version=1.19.0

**Comment by Frawless** [comment]
@strimzi-ci run tests profile=smoke test_only kube_version=1.19.0

**Comment by Frawless** [comment]
@strimzi-ci run tests profile=smoke test_only

**Comment by Frawless** [comment]
@strimzi-ci run tests profile=smoke test_only kube_version=1.19.0

**Comment by Frawless** [comment]
@strimzi-ci run tests profile=acceptance test_only kube_version=1.19.0

**Comment by Frawless** [comment]
@strimzi-ci run tests profile=acceptance,networkpolicies test_only kube_version=1.19.0

**Comment by Frawless** [comment]
@strimzi-ci run tests profile=networkpolicies test_only kube_version=1.19.0

**Comment by Frawless** [comment]
@strimzi-ci run tests profile=acceptance kube_version=1.19.0

**Comment by Frawless** [comment]
@strimzi-ci run tests profile=acceptance kube_version=1.19.0

**Comment by Frawless** [comment]
@strimzi-ci run tests profile=acceptance kube_version=1.19.0

**Comment by Frawless** [comment]
@strimzi-ci run tests profile=acceptance kube_version=1.19.0

**Comment by Frawless** [comment]
@strimzi-ci run tests profile=acceptance test_only

**Comment by Frawless** [comment]
@strimzi-ci run tests profile=acceptance test_only

File: systemtest/scripts/results_info.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ RESULTS_PATH=${1}
 TEST_CASE=${2}
 TEST_PROFILE=${3}
 BUILD_ID=${4:-0}
-OCP_VERSION=${5:-3}
+KUBE_VERSION=${5:-1.15.0}
```
Comment (scholzj) [inline systemtest/scripts/results_info.sh:7]: 1.16.0?

**Review by scholzj — APPROVED** [review]

**Comment by Frawless** [comment]
The failures on Jenkins are connected to env from my POV.

**Pull request merged**